src/View/Helper/LessHelper.php

```diff
@@ -147,14 +147,16 @@ public function less($less = 'styles.less', array $options = [], array $modifyVa
         }
 
         try {
-            $css = $this->compile($less, $options['cache'], $options['parser'], $modifyVars);
+        	$cache_dir = isset($options['parser']['cache_dir']) ?$options['parser']['cache_dir']:""; 
```

Comment:
bad indentation there + missing spaces in concatenation. BTW, this will be better inside a custom function, so we can properly test the cache behavior.